proposals/description/oa_cr_fix.ts

```diff
@@ -0,0 +1,34 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_x: ProposalDescription = {
+  title: 'OA CR Fixes',
+  commands: [
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'swapDeposit(address,address)',
+      arguments: ['{balancerLensBpt30Fei70Weth}', '{balancerLensBpt30Fei70WethFixed}'],
```

Comment:
now is a good time to rethink our naming conventions. The arbitrary suffix "fixed" is not a super scalable convention.

If it has the same ABI I'd prefer making the fixed one called `balancerLensBpt30Fei70Weth` and naming the old one "old" or 
"v1" or something

Reply:
Yeah would prefer version numbers over "fixed"

Reminds me of how I used to name files in folders for school assignments. "Assignment1Fixed", "Assignment1FixedFinal", "Assignment1FixedFinalFinalSubmitThis.doc"

Reply:
renamed the old one to `-Old`